weblab/templates/entities/entity_version.html

```diff
@@ -40,12 +41,17 @@ <h2 id="entityversionname">
             <a href="{{ entity|url_tag_version:version }}">Add tag</a>.
             <form method="POST" action="">
               Change visibility: {{ form.visibility }}
-              <img src="{% static 'img/info.png' %}" alt="help" title="{{ entity.VISIBILITY_HELP }}" />.
+              <img src="{% static 'img/info.png' %}" alt="help" title="{{ entity.VISIBILITY_HELP }}" />
               <span id="versionVisibilityAction"></span>
-            </form>
-          {% else %}
+              <!-- SK: put in permission -->
```

Comment:
Yeah, I think you want this bit guarded only by `{% if perms.create_experiment %}` rather than whether they can modify the entity itself.

Reply:
Done